stories/feed-input/feed-input.stories.js

```diff
@@ -5,86 +5,91 @@ export default {
-        <button class="fd-button sap-icon--feeder-arrow fd-feed-input__submit-button" aria-label="Submit" title="Submit" aria-disabled="false"></button>
+        <textarea class="fd-textarea fd-feed-input__textarea" id="textarea-3" aria-label="Feed" aria-required rows="3" placeholder="Post something here">With every line\ntext area grows upward\nuntil it reaches the max height</textarea>
+        <button class="fd-button sap-icon--feeder-arrow fd-feed-input__submit-button" aria-label="Send" title="Send" aria-disabled="false"></button>
     </div>
 `;
 
-inputGrowth.storyName = 'Multiple lines';
+inputGrowth.storyName = 'Multiline';
 inputGrowth.parameters = {
     docs: {
-        storyDescription: 'Every new line extends textarea height until maximum height is reached'
+        iframeHeight: 180,
+        storyDescription: 'Every new line extends textarea height until maximum height is reached. The multiline feed input displays a text field with multiple lines of text. When a new line is added to the field, the text area\'s height extends until the maximum height is reached.'
     }
 };
 
 export const overMaxHeight = () => `
     <div class="fd-feed-input" role="region" aria-label="Feed input over max height">
         <div class="fd-feed-input__thumb" style="background-image: url(http://lorempixel.com/460/620/people/7/)" role="img" title="Thumbnail"></div>
-        <textarea class="fd-textarea fd-feed-input__textarea" id="textarea-4" rows="18" aria-label="Feed" placeholder="Post something here">Over max height\nOver max height\nOver max height\nOver max height\nOver max height\nOver max height\nOver max height\nOver max height\nOver max height\nOver max height\nOver max height\nOver max height\nOver max height\nOver max height\nOver max height\nOver max height\nOver max height\nOver max height</textarea>
-        <button class="fd-button sap-icon--feeder-arrow fd-feed-input__submit-button" aria-label="Submit" title="Submit" aria-disabled="false"></button>
+        <textarea class="fd-textarea fd-feed-input__textarea" id="textarea-4" rows="18" aria-label="Feed message" aria-required placeholder="Post something here">Text\nText\nText\nText\nText\nText\nText\nText\nText\nText\nText\nText\nText\nText\nText\nText\nText\nText\nText\nText</textarea>
+        <button class="fd-button sap-icon--feeder-arrow fd-feed-input__submit-button" aria-label="Send" title="Send" aria-disabled="false"></button>
     </div>
 `;
 
-overMaxHeight.storyName = 'Over max height';
+overMaxHeight.storyName = 'Max height';
 overMaxHeight.parameters = {
     docs: {
-        storyDescription: 'Once textarea reaches the maximum height the textarea scroll is shown'
+        iframeHeight: 420,
+        storyDescription: 'The maximum feed input displays a text field with the maximum amount of text lines it can contain. Once the text field has reached the maximum height, a scrollbar appears to access the overflowing text.'
```

Comment:
Since you've decided on "Max height" as a title, I would just tweak this sentence to say "The max height feed input displays..." Make sure to use the title of the example/type of component in the description. :)

Reply:
fixed